src/lib.rs

```diff
@@ -7,6 +7,13 @@
 
 //! Helpers for writing Alfred script filter output
 //!
+//! Additionally the crate provides a way of checking for and updating
+//! your workflows automatically (Alfred 3 and above only).
+//!
+//! See [`updater`] module documentation for details and examples.
```

Comment:
I think this is working because in next line I have defined what `[updater]` is pointing to.
However I ran to similar issues when using nightly/stable. It seems rustdoc is moving to a new markdown backend and this is related to the transition.